.jenkins/caffe2/build.sh

```diff
@@ -65,7 +65,7 @@ if [[ "${BUILD_ENVIRONMENT}" == conda* ]]; then
   # headers are those in conda as well
   # This path comes from install_anaconda.sh which installs Anaconda into the
   # docker image
-  PROTOBUF_INCDIR=/opt/conda/include pip install "${ROOT_DIR}/third_party/onnx"
+  PROTOBUF_INCDIR=/opt/conda/include CMAKE_ARGS="${CMAKE_ARGS} -DONNX_NO_WERROR=ON" pip install "${ROOT_DIR}/third_party/onnx"
```

Comment:
Hmm this means we need to relax the checks in onnx, shouldn't ask users to pass additional flags to install onnx

Reply:
++